sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -155,6 +159,67 @@ dif_rstmgr_result_t dif_rstmgr_reset_info_clear(const dif_rstmgr_t *handle) {
   return kDifRstmgrOk;
 }
 
+dif_rstmgr_result_t dif_rstmgr_alert_info_set_enabled(
+    const dif_rstmgr_t *handle, dif_rstmgr_toggle_t state) {
+  if (handle == NULL) {
+    return kDifRstmgrBadArg;
+  }
+
+  mmio_region_t base_addr = handle->params.base_addr;
+
+  uint32_t enabled = (state == kDifRstmgrToggleEnabled) ? 0x1 : 0x0;
+
+  // This will clobber the `ALERT_INFO_CTRL.INDEX` field. However, the index
+  // field is only relevant during the crash dump read operation, and is
+  // set by the caller and not the hardware, so it is safe to clobber it.
+  mmio_region_write32(base_addr, RSTMGR_ALERT_INFO_CTRL_REG_OFFSET, enabled);
```

Comment:
I wonder if I should just write only to the bit `0` by using the `bitfield` library? It will involve couple additional calls, and Read-Write-Modify arrangement, but potentially makes the implementation more robust in case this register layout changes.

Reply:
I don't think this is an issue, but it would be good to get @tjaychen to confirm, and we may need a note about this in the documentation on this function in the header.

Reply:
yep this looks okay to me!